managing_providers/_topics/embedded_workflows.md

```diff
@@ -0,0 +1,339 @@
+
+Refresh a targeted repository:
+
+1.  Browse to the menu: **Automation** > **Embedded Workflows** > **Repositories**.
+
+2.  Click a repository.
+
+3.  Click **Configuration**, then ![Refresh this Repository](../images/2003.png) (**Refresh this Repository**).
+
+Alternately, you can refresh some or all repositories from the list view:
+
+1.  Browse to the menu: **Automation** > **Embedded Workflows** > **Repositories**.
+
+2.  Check those repositories to refresh. Click **Check All** to select all repositories.
+
+3.  Click **Configuration**, then ![Refresh Selected Workflow Repositories](../images/2003.png)(**Refresh Selected Workflow Repositories**).
+
+### Adding Credentials
+
+Credentials are used by {{ site.data.product.title_short }} for any authentication that is required when running embedded workflows, and when importing workflows from a version control system.
+
+{{ site.data.product.title_short }} can store credentials that are used by embedded workflows. Credentials that are saved in {{ site.data.product.title_short }} are matched and executed with a workflow when it runs.
+
+1.  Browse to the menu: **Automation** > **Embedded Workflows** > **Credentials**.
+
+2.  Click **Configuration**, then ![Add New Credential](../images/1862.png) (**Add New Credential**).
+
+3.  Provide a **Name** for the credential.
+
+4.  Select the **Credential Type**. More fields appear depending on the type chosen.
+
+5.  Click **Add**.
+
+To view the added credentials, return to the **Automation** > **Embedded Workflows** > **Credentials** page. The list of credentials is displayed. To view a summary of details for a credential, click the credential to open the *Summary* page for that credential.
+
+If you need to edit a credential, select the credential from the list of credentials. Then, click **Configuration** > **Edit this Credential**. After you complete your edits, click **Save**.
+
+If you need to delete a credential, select the credential from the list of credentials. Then, click **Configuration** > **Remove selected Credentials from Inventory**.
+
+#### Credential Types
+
+Each credential type that is used by {{ site.data.product.title_short }} for embedded workflows is detailed in the following sections.
+
+##### SCM
+
+SCM (source control Management) credentials are used with Projects to clone and update local source code Git repositories. Source Control credentials have several attributes that can be configured:
+
+- **Username**: The username to use in conjunction with the source control system.
+
+- **Password**: The password to use in conjunction with the source control system.
+
+- **Private key passphrase**: If the SSH private key used is protected by a passphrase, you can configure a key passphrase for the private key.
+
+- **Private Key**: Copy or drag-and-drop the actual SSH Private Key to be used to authenticate the user to the source control system by using SSH.
+
+### Authoring a workflow
```

Comment:
It looks like we lost the "##### Workflows` credential type here, the only type I see is `SCM`

Reply:
All other credential types were removed based on previous feedback. Should any be added or the section heading/sentence removed and rewritten to merge into adding credentials section.